workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -170,4 +170,15 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
       }
     })
   );
+  router.get(
+    '/authorization/identitypermissions/identity',
```

Comment:
In #733 it was suggested to change to `/authorization/permissions/*`

Reply:
It was a suggestion. Not sure if there is any reason why permissions would get further overloaded by some other entity.